### ISSUE TITLE
[Feature]: Allow returning YAML instead of json

### ISSUE THREAD

**Issue by mdornseif** [ioBroker/ioBroker.simple-api#152]
**Is your feature request related to a problem? Please describe.**
I have to parse responses with a very minimal pattern matching engine on embedded hardware. JSON is hard to parse with that. YAML would be much easier to match.

**Describe the solution you'd like**
Allow http://ipaddress:8087/objects?pattern=system.adapter.admin.0.*&yaml which would return a YAML document instead of a JSON document.

**Describe alternatives you've considered**
Building a JSON-to-YAML proxy.

**Comment by Apollon77** [comment]
Sorry but yaml is no standard in API communicationm just for config files ...
